apidef/migration.go

```diff
@@ -78,3 +78,37 @@ func (a *APIDefinition) MigrateVersioning() (versions []APIDefinition, err error
 
 	return
 }
+
+func (a *APIDefinition) MigrateCachePlugin() (err error) {
+	if a.VersionDefinition.Enabled || len(a.VersionDefinition.Versions) != 0 {
+		return errors.New("not migratable - new versioning is enabled")
+	}
+
+	if a.VersionData.NotVersioned && len(a.VersionData.Versions) > 1 {
+		return errors.New("not migratable - if not versioned, there should be 1 version info in versions map")
```

Comment:
This part is already implemented in versioning function. No need to call here again